docs/reference/commandline/plugin_enable.md

```diff
@@ -21,7 +21,8 @@ Usage:  docker plugin enable PLUGIN
 Enable a plugin
 
 Options:
-      --help   Print usage
+      --help          Print usage
+      --timeout int   HTTP client timeout (in seconds)
```

Comment:
Duration is not necessary here. This was already discussed at https://github.com/docker/docker/pull/28595#discussion_r88989260